site/content/entry/yuzu-progress-report-dec-2021/index.md

```diff
@@ -0,0 +1,371 @@
+or how to optimize it.
+
+Link time optimizations, on the other hand, take into consideration all the functions in the project.
+The linker, thus, is able to perform the same optimizations as the compiler, but more efficiently, as it is aware of the contents of all the functions defined in the project.
+This comes at a price, since the process needs more memory and takes more time to finish, but it guarantees that the released binaries perform better.
+
+Along with this work, we considered enforcing [SSE4.2](https://en.wikipedia.org/wiki/SSE4#SSE4.2) support, improving performance but making yuzu incompatible with 12 year 
+old CPUs like the Core 2 Duo and Phenom II or older.
+While the performance results were positive, the developers are still debating about reducing CPU compatibility..
+
+When you open yuzu, the emulator has to take some time to measure the [RDTSC frequency](https://en.wikipedia.org/wiki/Time_Stamp_Counter), a way to measure the clock 
+speed of the CPU.
+Due to a bit of bloat in the previous implementation, 3 full seconds were needed to complete the operation.
+Morph [rewrote the whole code section](https://github.com/yuzu-emu/yuzu/pull/7494) and now only 0.2 seconds (200 milliseconds) are needed to get results as accurate as
+before, considerably reducing the boot times of the emulator itself.
+
+As previously said, german77 continues to work in making `Ring Fit Adventure` playable. 
+He has [stubbed the](https://github.com/yuzu-emu/yuzu/pull/7524) `SetNpadCaptureButtonAssignment`, `ClearNpadCaptureButtonAssignment`, `ListAlarmSettings`, and 
+`Initialize` services, and [added support](https://github.com/yuzu-emu/yuzu/pull/7525) for the `notif:a` service.
+
+With all his changes, the current Early Access build when writting this article can boot and play the first stage of the game:
+
+{{< imgs
+	"./ringfit.mp4| Ring Fit Adventure"
+  >}}
+
+[Tatsh](https://github.com/Tatsh) [added NSP and XCI file association to Linux](https://github.com/yuzu-emu/yuzu/pull/7609).
+Thanks!
+
+[Tachi107](https://github.com/Tachi107) [updated cubeb](https://github.com/yuzu-emu/yuzu/pull/7527) and removed now deprecated functions.
+Cleaner is always better, thanks!
+
+[heinermann](https://github.com/heinermann) fixed a crash that would occur when the 
+[emulation was paused and the window was out of focus](https://github.com/yuzu-emu/yuzu/pull/7506). 
+Thank you!
+
+[jbeich](https://github.com/jbeich) changed the building configuration so that 
+[VA-API, one of the video decoding APIs of Linux, is enabled on Unix systems](https://github.com/yuzu-emu/yuzu/pull/7602), allowing the users who want to build targeting 
+BSD or other Unix-based systems to use hardware acceleration for video decoding.
+
+This is just one of several PRs jbeich wrote to help yuzu work on BSD systems, thank you for your contributions!
```

Comment:
```suggestion
[Tatsh](https://github.com/Tatsh) [added NSP and XCI file association to Linux](https://github.com/yuzu-emu/yuzu/pull/7609).
Thanks!

[Tachi107](https://github.com/Tachi107) [updated cubeb](https://github.com/yuzu-emu/yuzu/pull/7527) and removed now deprecated functions.
Cleaner is always better, thanks!

[heinermann](https://github.com/heinermann) fixed a crash that would occur when the 
[emulation was paused and the window was out of focus](https://github.com/yuzu-emu/yuzu/pull/7506). 
Thank you!

[jbeich](https://github.com/jbeich) changed the building configuration so that 
[VA-API, one of the video decoding APIs of Linux, is enabled on Unix systems](https://github.com/yuzu-emu/yuzu/pull/7602), allowing the users who want to build targeting 
BSD or other Unix-based systems to use hardware acceleration for video decoding.

This is just one of several PRs jbeich wrote to help yuzu work on BSD systems, thank you for your contributions!
```
Going forward, let's not thank each contributor individually as it interrupts the flow of the article. Let's do a general thanks at the bottom of the article in the sign-off, and if we're continuing to get first-time contributors, we can have a section where we thank them together.

Reply:
Roger that.